src/providers/push/fcm/fcm.provider.ts

```diff
@@ -33,6 +33,7 @@ export class FcmProvider implements Provider<FcmNotification> {
     }
 
     if (message.receiver.to.length > 500) {
+      //NOSONAR
```

Comment:
this will still be a code smell as the //nosonar is on the next line 
same define a constant

Reply:
done changes